streams/src/main/java/org/apache/kafka/streams/Consumed.java

```diff
@@ -24,10 +24,29 @@
 
 /**
  * The {@code Consumed} class is used to define the optional parameters when using {@link StreamsBuilder} to
- * build instancs of {@link KStream}, {@link KTable}, and {@link GlobalKTable}.
+ * build instances of {@link KStream}, {@link KTable}, and {@link GlobalKTable}.
+ * <p>
+ * For example, you can read an topic as {@link KStream} with custom timestamp extractor and specify the corresponding
+ * key and value serdes like:
+ * <pre>{@code
+ * StreamsBuilder builder = new StreamsBuilder();
+ * KStream<String, Long> stream = builder.stream(
+ *   "topicName",
+ *   Consumed.with(Serdes.String(), Serdes.Long())
+ *           .withTimestampExtractor(new LogAndSkipOnInvalidTimestamp()));
+ * }</pre>
+ * Similar, you can read a topic as {@link KTable} with custom "auto.offset.reset" strategy and force a state store
+ * {@link org.apache.kafka.streams.kstream.Materialized materialization} to access the content via Interactive Queries API:
```

Comment:
Suggestion: We already say "Kafka Streams API", "DSL", and "Processor API".  That's already a lot of "APIs" even though it's all about the same thing.  I wouldn't label interactive queries to be yet another API.

I'd just say:
> ... to access the content via interactive queries: